platform-sdk/docs/proposals/template.md

```diff
@@ -12,6 +12,7 @@ Give a 1-3 sentence summary of the capability in the design proposal.
 | Functional Impacts | Services, DevOps, Mirror Node, etc. |
 | Related Proposals  | Proposal-1, Proposal-2              |
 | HIPS               | HIP-1, HIP-2,                       |
+| State              | Withdrawn by template-2.md          |
```

Comment:
Two things:
1. I am confused by this wording. A proposal is "withdrawn by" anything
2. I disagree with adding this state. The state is tracked in the Design Proposal Project and it is not good to track it in multiple places.

Reply:
ok, will remove